contribs/gmf/examples/mobilebackgroundlayerselector.html

```diff
@@ -56,9 +56,9 @@
     directive to select the map background layer.</p>
     <div>
         <span>Select base layer:</span>
-        <gmf-mobile-background-layer-selector
-          gmf-mobile-background-layer-selector-map="::ctrl.map">
-        </gmf-mobile-background-layer-selector>
+        <gmf-mobilebackgroundlayerselector
```

Comment:
both hurts my eyes, but this is consistent with other files...

I was thinking that this was in the guidlines but it seems that that was only a conversation with elemoine...